resources/js/features/games/components/TopAchieversMainRoot/TopAchieversList/TopAchieversList.tsx

```diff
@@ -0,0 +1,85 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  BaseTable,
+  BaseTableBody,
+  BaseTableCell,
+  BaseTableHead,
+  BaseTableHeader,
+  BaseTableRow,
+} from '@/common/components/+vendor/BaseTable';
+import { PlayerBadgeIndicator } from '@/common/components/PlayerBadgeIndicator';
+import { PlayerBadgeLabel } from '@/common/components/PlayerBadgeLabel';
+import { UserAvatar } from '@/common/components/UserAvatar';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { AwardType } from '@/common/utils/generatedAppConstants';
+import { formatDate } from '@/common/utils/l10n/formatDate';
+import { cn } from '@/utils/cn';
+
+export const TopAchieversListContainerTestId = 'top-achievers-list';
+
+export const TopAchieversList: FC = () => {
+  const { paginatedUsers } = usePageProps<App.Platform.Data.GameTopAchieversPageProps>();
+
+  const { t } = useTranslation();
+
+  if (!paginatedUsers.items.length) {
+    return null;
+  }
+
+  return (
+    <BaseTable
+      containerClassName={cn(
+        'overflow-auto rounded-md border border-neutral-700/80 bg-embed',
+        'light:border-neutral-300 lg:overflow-visible lg:rounded-sm',
+        '',
+        '',
+      )}
+    >
+      <BaseTableHeader>
+        <BaseTableRow className="do-not-highlight">
+          <BaseTableHead>{t('Rank')}</BaseTableHead>
+          <BaseTableHead>{t('User')}</BaseTableHead>
+          <BaseTableHead>{t('Progress')}</BaseTableHead>
+        </BaseTableRow>
+      </BaseTableHeader>
+
+      <BaseTableBody>
+        {paginatedUsers.items.map((achiever) => (
+          <BaseTableRow key={achiever.user.displayName}>
+            <BaseTableCell>{achiever.rank}</BaseTableCell>
+
+            <BaseTableCell>
+              <UserAvatar {...achiever.user} size={32} />
+            </BaseTableCell>
+
+            <BaseTableCell>
+              {achiever.badge ? (
+                <div>
+                  {achiever.badge.awardType == AwardType.Mastery && (
+                    <span>{formatDate(achiever.badge.awardDate, 'lll')}</span>
+                  )}
+
+                  <div className={cn('flex items-center gap-1')}>
+                    {achiever.badge.awardType == AwardType.GameBeaten && (
+                      <span>
+                        {achiever.score}
+                        <span className="text-muted">{' - '}</span>
+                      </span>
+                    )}
+
+                    <PlayerBadgeIndicator playerBadge={achiever.badge} className="mt-px" />
+                    <PlayerBadgeLabel playerBadge={achiever.badge} variant="muted-group" />
+                  </div>
+                </div>
+              ) : (
+                <span>{achiever.score}</span>
```

Comment:
Similar feedback here as the other `score` values - this should probably pass through the number formatter.

Reply:
done